### ISSUE TITLE
Select quality and file format in youtube-dl

### ISSUE THREAD

**Issue by xaionaro** [schollz/playlistfromsong#6]
Hello, again :)

If I understand correctly `playlistfromsong` recodes music into mp3. Is there a way to prevent it (and use another format)? Or at least is there a way to select a bitrate? Or this is useless because source of the music already has mp3/192kbps-like quality?

I can see that some of sources uses OPUS. That is much better than mp3, IMHO.

Best regards, Dmitry.

**Comment by xaionaro** [comment]
Ok, seems I just need to fix lines here:

https://github.com/schollz/playlistfromsong/blob/4786c9d120748d7bde06fb4e4353e709cbbe2f69/playlistfromsong/__main__.py#L71

Sorry for premature issue. This's enough for me. Should I close the issue? :)

**Comment by schollz** [comment]
No, lets leave it up! I think this could be a good feature. Its easy to add an argument for doing this easier in the future.

**Comment by rachmadaniHaryono** [comment]
hi @xaionaro i create #10 so you can set the codec and quality you want. can you test it.

just open the config file `playlistfromsong config -o` and add following key

    ffmpeg_codec: mp3
    ffmpeg_quality: 192

change those `mp3` and `192` into whatever codec and quality you want.

**Comment by xaionaro** [comment]
@rachmadaniHaryono, thanks :). Is there a way to just disable the recoding?

**Comment by rachmadaniHaryono** [comment]
with program config, youtube-dl option can be put on it, and  user can change it himself. 

but i think that is too much. 

opinion @schollz ?

**Comment by schollz** [comment]
I'm not sure I understand, you want to disable the recoding to download OPUS directly @xaionaro ?

**Comment by rachmadaniHaryono** [comment]
@schollz I think @xaionaro mean decoding.

This can be done by removing postsprocessors key in YouTube-dl option,so YouTube dl can install the best audio (which can be opus)  and don't decode it to mp3
